commit b76f96ecb8d20f1b34cd487b195867e0948cf237
Author: Phil Elwell <phil@raspberrypi.org>
Date:   Wed Oct 25 09:20:56 2017 +0100

    scripts: Update mkknlimg, just in case
    
    With the removal of the vc_cma driver, mkknlimg lost an indication that
    the user had built a downstream kernel. Update the script, adding a few
    more key strings, in case it is still being used.
    
    Note that mkknlimg is now deprecated, except to tag kernels as upstream
    (283x), and thus requiring upstream DTBs.
    
    See: https://github.com/raspberrypi/linux/issues/2239
    
    Signed-off-by: Phil Elwell <phil@raspberrypi.org>

diff --git a/scripts/mkknlimg b/scripts/mkknlimg
index 84be2593ec1d..d72e15224232 100755
--- a/scripts/mkknlimg
+++ b/scripts/mkknlimg
@@ -68,18 +68,15 @@ if (! -r $kernel_file)
 
 my $wanted_strings =
 {
-	'bcm2708_fb' => FLAG_PI | FLAG_270X,
 	'brcm,bcm2835-mmc' => FLAG_PI,
 	'brcm,bcm2835-sdhost' => FLAG_PI,
-	'brcm,bcm2708-pinctrl' => FLAG_PI | FLAG_DTOK,
 	'brcm,bcm2835-gpio' => FLAG_PI | FLAG_DTOK,
-	'brcm,bcm2708' => FLAG_PI | FLAG_DTOK | FLAG_270X,
-	'brcm,bcm2709' => FLAG_PI | FLAG_DTOK | FLAG_270X,
+	'brcm,bcm2708-fb'   => FLAG_PI | FLAG_DTOK | FLAG_270X,
+	'brcm,bcm2708-usb'  => FLAG_PI | FLAG_DTOK | FLAG_270X,
 	'brcm,bcm2835' => FLAG_PI | FLAG_DTOK | FLAG_283X,
 	'brcm,bcm2836' => FLAG_PI | FLAG_DTOK | FLAG_283X,
+	'brcm,bcm2837' => FLAG_PI | FLAG_DTOK | FLAG_283X,
 	'of_cfs_init' => FLAG_DTOK | FLAG_DDTK,
-	'vc_cma_init' => FLAG_PI | FLAG_270X,
-	'vc-mem' => FLAG_PI | FLAG_270X,
 };
 
 my $res = try_extract($kernel_file, $tmpfile1);
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1144_b76f96ecb8d20f1b34cd487b195867e0948cf237.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1144_b76f96ecb8d20f1b34cd487b195867e0948cf237.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
